photon-ml/src/main/scala/com/linkedin/photon/ml/sampler/BinaryClassificationDownSampler.scala

```diff
@@ -29,8 +30,7 @@ import scala.util.hashing.byteswap64
  * @param downSamplingRate The down sampling rate
  */
 protected[ml] class BinaryClassificationDownSampler(downSamplingRate: Double) extends DownSampler with Serializable {
-
-  require((downSamplingRate > 0D) && (downSamplingRate <= 1D), s"Invalid down-sampling rate: $downSamplingRate")
+  require((downSamplingRate > 0D) && (downSamplingRate < 1D), s"Invalid down-sampling rate: $downSamplingRate")
```

Comment:
Minor issue, but why make this change? ```downSamplingRate``` can be 1D sometimes, and actually the default for ```downSamplingRate``` is ```1D```.